docs/product-installation-guide/src/main/asciidoc/clustering-artemis-activate-proc.adoc

```diff
@@ -10,5 +10,5 @@ Elasticsearch installed on the main node of the cluster
 . Add the following properties to the `EAP_HOME/standalone/configuration/standalone.xml` file:
 * Add `<socket-binding name="activemq" port="61616"/>` to the socket-binding-group element.
 * Add `<remote-acceptor name="activemq-acceptor" socket-binding="activemq"/>` in `<server name="default">` of the messaging-activemq subsytem element.
-. Complete the steps in <<clustering-dm-install-proc>> on each node of the cluster.
+. Complete the steps in <<clustering-dm-install-proc_{context}>> on each node of the cluster.
```

Comment:
since this refers to on premise deployments don't we want to use domain mode of EAP or is this already deprecated/removed?